accepted/2021/statics-in-interfaces/README.md

```diff
@@ -789,90 +814,129 @@ namespace System
+
+        // The following methods are approved but not yet implemented in the libraries
 
+        static abstract TSelf LogP1(TSelf x);
+
+        static abstract TSelf Log2P1(TSelf x);
+
+        static abstract TSelf Log10P1(TSelf x);
+    }
+
+    public interface IPowerFunctions<TSelf>
+        where TSelf : IPowerFunctions<TSelf>
+    {
+        static abstract TSelf Pow(TSelf x, TSelf y);
+    }
+
+    public interface IRootFunctions<TSelf>
+        where TSelf : IRootFunctions<TSelf>
+    {
         static abstract TSelf Cbrt(TSelf x);
 
-        static abstract TSelf Ceiling(TSelf x);
+        static abstract TSelf Sqrt(TSelf x);
 
-        // CopySign is likely more general purpose and may apply to any signed number type
-        // It may even be applicable to unsigned numbers where it simply returns x, for convenience
+        // The following methods are approved but not yet implemented in the libraries
 
-        static abstract TSelf CopySign(TSelf x, TSelf y);
+        static abstract TSelf Root(TSelf x, TSelf n);
+    }
 
-        static abstract TSelf Cos(TSelf x);
+    public interface ITrigonometricFunctions<TSelf>
+        where TSelf : ITrigonometricFunctions<TSelf>
+    {
+        static abstract TSelf Acos(TSelf x);
 
-        static abstract TSelf Cosh(TSelf x);
+        static abstract TSelf Asin(TSelf x);
 
-        static abstract TSelf Exp(TSelf x);
+        static abstract TSelf Atan(TSelf x);
 
-        static abstract TSelf Floor(TSelf x);
+        static abstract TSelf Atan2(TSelf y, TSelf x);
 
-        static abstract TSelf FusedMultiplyAdd(TSelf x, TSelf y, TSelf z);
+        static abstract TSelf Cos(TSelf x);
 
-        static abstract TSelf IEEERemainder(TSelf x, TSelf y);
+        static abstract TSelf Sin(TSelf x);
 
-        // IEEE defines the result to be an integral type, but not the size
+        static abstract TSelf Tan(TSelf x);
 
-        static abstract TInteger ILogB<TInteger>(TSelf x)
-            where TInteger : IBinaryInteger<TInteger>;
+        // The following methods are approved but not yet implemented in the libraries
 
-        static abstract TSelf Log(TSelf x);
+        static abstract TSelf AcosPi(TSelf x);
 
-        static abstract TSelf Log(TSelf x, TSelf newBase);
+        static abstract TSelf AsinPi(TSelf x);
 
-        static abstract TSelf Log2(TSelf x);
+        static abstract TSelf AtanPi(TSelf x);
 
-        static abstract TSelf Log10(TSelf x);
+        static abstract TSelf Atan2Pi(TSelf y, TSelf x);
 
-        static abstract TSelf MaxMagnitude(TSelf x, TSelf y);
+        static abstract TSelf CosPi(TSelf x);
 
-        static abstract TSelf MinMagnitude(TSelf x, TSelf y);
+        static abstract TSelf SinPi(TSelf x);
 
-        static abstract TSelf Pow(TSelf x, TSelf y);
+        static abstract TSelf TanPi(TSelf x);
+    }
+
+    public interface IFloatingPoint<TSelf>
+        : ISignedNumberBase<TSelf>
+        where TSelf : IFloatingPoint<TSelf>
```

Comment:
`IFloatingPoint` now covers the APIs common between `half`, `float`, `double`, and `decimal`. This should likewise make it sufficient to be implemented by user-defined floating-point types.